Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -67,7 +67,7 @@ class ApolloCodegenTests: XCTestCase {
     let namespace = "ANameSpace"
     let prefix = "MyPrefix"
     
-    let options = ApolloCodegenOptions(codegenEngine: .swiftExperimental,
```

Comment:
for what it's worth I think it's worth keeping the `swiftExperimental` case since it's still eventually going to be used, just for a completely different setup

Reply:
It comes back in the other RFC-focused branch. My opinion was that if it stayed in `main` then it should blow up, aka `fatalError`, because it does nothing at this point. Easier to nuke from `main` and bring it back where we actually use it.

Reply:
eh, fair enough. I'm always more in favor of just leaving it but I see your point too.